doc(bpmn_visu): documentation for Fred on how to find and use case/process visu (#2083)
bonitasoft/bonita-doc: modules/pages-and-forms/pages/create-or-modify-a-page.adoc

Only Bonita pages created with the UI Designer can be customized in the UI Designer. Indeed, as Bonita applications have evolved in several phases, so did the frameworks in which their pages have been made.

To customize a page provided by Bonita:

. From Bonita Studio coolbar, click on the *_Applications_* icon, and open the *_Administrator Application_*

> . Go to the *_Resources_* menu, find the page, and make sure it’s done with the UI Designer

Reviewer: ```suggestion
. Go to the *_Resources_* menu, find the page, and and check its description to make sure it is a page done with Bonita UI Designer
```

. Export the page

> . From Bonita Studio coolbar, click on the *_UI Designer_* icon to open the UI Designer

Reviewer: ```suggestion
. From Bonita Studio coolbar, click on the *_UI Designer_* icon to open Bonita UI Designer
```

. Import the page in the UI Designer

> Note that by doing so, if the page embed fragments and custom widgets, they are added to the UI Designer palette.

Reviewer: ```suggestion
Note that by doing so, if the page embed fragments and custom widgets, they are also imported and added to the UI Designer palette.
```

== Create a fragment

> Use the same sequence of steps than to create a page.

Reviewer: ```suggestion
Use the same sequence of steps as to create a page.
```
